.vscode/tasks.json

```diff
@@ -0,0 +1,28 @@
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "type": "shell",
+            "label": "C/C++: g++ build active file",
+            "command": "/usr/bin/g++",
+            "args": [
+                "-std=gnu++14",
```

Comment:
Will `-std=c++17` also work?

Reply:
I am adding update. I see. But for similar environment, I committed this dot file. even this is not allowed?